userguide/tutorials/aws-standalone.adoc

```diff
@@ -0,0 +1,250 @@
+= Installation via Our Standalone AMI
+
+== One Node Deployement 
+
+In order to get started as quick as possible we offer an AMI that also bundles the mysql database as part of the same image. This makes it very easy to get started, as this only requires the click of a button.
+
+Running Kill Bill on AWS using our official AMI lets you deploy Kill Bill, Kaui and MySQL in a single EC2 instance. It is perfect for test environments or for small production deployments where high availability or scale are not a concern.
+
+After launching the AMI you should have a setup that looks like the following:
+
+image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/standalone_deployment.png[align=center]
+
+== Multi Nodes Deployments
+
+For production deployments, allocating a separate RDS database instance and allowing multiple nodes to handle traffic behind a load balancer is the recommended option.
+
+The following list of pre-requisites is needed for this option to work:
+
+1. Setup of the RDS instance with the right Kill Bill schema
+2. Setup of the ELB with the correct security groups
+
+
+Note that we also offer the ability to deploy the whole stack using https://github.com/killbill/killbill-docs/blob/v3/userguide/tutorials/aws-cf.adoc[CloudFormation templates] to make it easier.
+
+
+This can be represented with the figure below:
+
+image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/standalone_nodb_deployment.png[align=center]
+
+
+
+== Setup Steps
```

Comment:
In terms of docs organization, this section is shared for single- and multi-nodes, right? If so, we should probably have a `== Overview` section above with these two mentioned sections (as `===`), so that this one stands out.

Reply:
Yes, good advice, will fix